avocado_vt/test.py

```diff
@@ -259,27 +259,28 @@ def setUp(self):
             self.__status = details
         finally:
             # Clean libvirtd debug logs if the test is not fail or error
-            if(self.params.get("vm_type") == 'libvirt' and
-               self.params.get("enable_libvirtd_debug_log", "yes") == "yes"):
-                libvirtd_log = self.params["libvirtd_debug_file"]
-                if("TestFail" not in str(sys.exc_info()[0]) and
-                   "TestError" not in str(sys.exc_info()[0])):
-                    if libvirtd_log and os.path.isfile(libvirtd_log):
-                        logging.info("cleaning libvirtd logs...")
-                        os.remove(libvirtd_log)
-                else:
-                    # tar the libvirtd log and archive
-                    logging.info("archiving libvirtd debug logs")
-                    from virttest import utils_package
-                    if utils_package.package_install("tar"):
-                        archive = os.path.join(libvirtd_log.strip(os.path.basename(libvirtd_log)),
-                                               "libvirtd.tar.gz")
-                        cmd = "tar -zcf %s -P %s" % (archive, libvirtd_log)
-                        if process.system(cmd) == 0:
+            if self.params.get("libvirtd_log_cleanup", "no") == "yes":
+                if(self.params.get("vm_type") == 'libvirt' and
+                   self.params.get("enable_libvirtd_debug_log", "yes") == "yes"):
+                    libvirtd_log = self.params["libvirtd_debug_file"]
+                    if("TestFail" not in str(sys.exc_info()[0]) and
+                       "TestError" not in str(sys.exc_info()[0])):
```

Comment:
this check can be simply added here ```
if("TestFail" not in str(sys.exc_info()[0]) and
                "TestError" not in str(sys.exc_info()[0]) and (self.params.get("libvirtd_log_cleanup", "no") == "yes")):```

Reply:
ah, okies, you basically need a plain libvirtd.log in all cases and do not want a archive at all, fine,
got it, but let us keep it `yes` bydefault as we do not want our test env to be flooded(consume more space) with lot of libvirtd logs, that's the reason we introduced log files cleanup and and archive in first place.

we had hit issues due to space, as some test can get hung/run for hours can easily lead to GBs of logs.